sparkmagic/example_config.json

```diff
@@ -65,5 +65,6 @@
   "livy_server_heartbeat_timeout_seconds": 0,
   "heartbeat_retry_seconds": 10,
 
-  "server_extension_default_kernel_name": "pysparkkernel"
+  "server_extension_default_kernel_name": "pysparkkernel",
+  "authentication_cookie_name": "hadoop.auth"
```

Comment:
@aggFTW There will be another PR raised for changes related to ldap authentication. I will add proper detail in README for livy conf.